contracts/auction-handler/FastLaneAuctionHandler.sol

```diff
@@ -372,6 +378,55 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents {
         emit RelayFeeCollected(_payor, block.coinbase, msg.value);
     }
 
+    function paySpecificValidatorFee(address _validator) external payable nonReentrant {
+        // TODO: block this when _validator == block.coinbase? 
+        if (msg.value == 0) revert RelayValueIsZero();
+        validatorsBalanceMap[_validator] += msg.value;
+        validatorsTotal += msg.value;
+        emit RelayFeeCollected(_validator, block.coinbase, msg.value);
+    }
```

Comment:
I don't think we need to block `_validator` being `block.coinbase` (we don't block it in `payValidatorFee`).

To stay consistent with the `payValidatorFee` function, I suggest adding a `_payor` param, and corrected values emitted by `RelayFeeCollected`.

Another suggestion would be dropping the `_payor` param for both `payValidatorFee` and `paySpecificValidatorFee` functions, and set the `payor` param of the `RelayFeeCollected` event to `msg.sender`.

```suggestion
    function paySpecificValidatorFee(address _payor, address _validator) external payable nonReentrant {
        if (msg.value == 0) revert RelayValueIsZero();
        validatorsBalanceMap[_validator] += msg.value;
        validatorsTotal += msg.value;
        emit RelayFeeCollected(_payor, _validator, msg.value);
    }
```